share/rocm/cmake/ROCMInstallTargets.cmake

```diff
@@ -146,6 +156,11 @@ function(rocm_install_targets)
         set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_INCLUDEDIR})
     endif()
 
+    if (PARSE_PRIVATE)
+        set(BIN_INSTALL_DIR ${BIN_INSTALL_DIR}/${PROJECT_NAME})
+        set(LIB_INSTALL_DIR ${LIB_INSTALL_DIR}/${PROJECT_NAME})
```

Comment:
I dont know if it makes sense to have a private lib directory. Either way, I think this would point to the same directory.